src/averageAges.js

```diff
@@ -20,28 +20,64 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const menLength = people.filter(person => person.sex === 'm').length;
+
+  const menCenturyLength = people.filter(person =>
+    person.sex === 'm' && Math.ceil(person.died / 100) === century).length;
+
+  const menAgeSum = people.reduce((acc, person) =>
+    person.sex === 'm' ? acc + (person.died - person.born) : acc + 0, 0);
```

Comment:
```suggestion
    person.sex === 'm'
      ? acc + (person.died - person.born)
      : acc + 0
    , 0);
```
Тернарники тоже переносим на новый ряд. Посмотри, так намного нагляднее. И смысл acc + 0. Давай тогда сразу acc + 9999 - 9999 * (1 / 0,5 * 2)

Reply:
Смысл того, что делает сумму, а написать так чтобы ничего не добавляет это + 0. Разве есть другой вариант написания?